tests/helpers.py

```diff
@@ -0,0 +1,50 @@
+"""Module to help to create tests."""
```

Comment:
Same suggestion as the other PR: move the most commonly used functions to kytos core.